src/data/set/intervals.lean

```diff
@@ -61,10 +63,12 @@ def Ioi (a : α) := {x | a < x}
 @[simp] lemma left_mem_Ico : a ∈ Ico a b ↔ a < b := ⟨λ h, h.2, λ h, ⟨le_refl _, h⟩⟩
 @[simp] lemma left_mem_Icc : a ∈ Icc a b ↔ a ≤ b := ⟨λ h, h.2, λ h, ⟨le_refl _, h⟩⟩
 @[simp] lemma left_mem_Ioc : a ∈ Ioc a b ↔ false := ⟨λ h, lt_irrefl a h.1, λ h, false.elim h⟩
+@[simp] lemma left_mem_Ici : a ∈ Ici a := by simp
```

Comment:
If `simp` solves this lemma, is there really a point of adding it to the simp-set?

Reply:
The same is true for the old lemmas in this paragraph as well. Should I remove `@[simp]` from all of them?

Reply:
Oh, if all of these are provable by `simp`, then I would indeed replace the proofs with `by simp` and remove the `simp` attributes.

Reply:
@fpvandoorn It turns out that I was wrong: `le_refl` and `lt_irrefl` have no `@[simp]` attributes, so `by simp` doesn't solve other lemmas.